docs/UserGuide.md

```diff
@@ -89,57 +92,65 @@ Adds a student to the address book.
 **Output:**
 ![addResult.jpeg](images/addResult.jpeg)
 
-<box type="important" header="#### Constraints">
+<box type="important">
 
-1. **SCHEDULE** must be in the format of `DAY_OF_THE_WEEK`-`START_TIME`-`END_TIME`
-2. **DAY_OF_THE_WEEK** includes `Monday` `Tuesday` `Wednesday` `Thursday` `Friday` `Saturday` `Sunday`
-3. **START_TIME** and <b>END_TIME</b> are represented as `HHmm`.
-4. **PHONE_NUMBER** should be 8 digits that starts with 6, 8 or 9.
-5. **RATE**, **PAID** and **OWED** must be at least 0 with at most 2 decimal places.
+##### Constraints
+
+* **SCHEDULE** must be in the format of `DAY_OF_THE_WEEK`-`START_TIME`-`END_TIME`
+* **DAY_OF_THE_WEEK** includes `Monday` `Tuesday` `Wednesday` `Thursday` `Friday` `Saturday` `Sunday`
+* **START_TIME** and <b>END_TIME</b> are represented as `HHmm`.
+* **PHONE_NUMBER** should be 8 digits that starts with 6, 8 or 9.
+* **RATE**, **PAID** and **OWED** must be at least 0 with at most 2 decimal places.
       <i>Example: </i> `12.00`, `0.0` or `7`
-6. **SUBJECT** should only be:
+* **SUBJECT** should only be
 `Economics`  `Literature`  `Music`  `Biology`  `Chemistry`  `Science`  
-`English`  `Chinese`  `Malay` `Tamil`  `Mathematics`  `History`  `Geography`  `Physics`  `GP`
+`English`  `Chinese`  `Malay` `Tamil`  `Mathematics`  `History`  `Geography`  `Physics` and `GP`
 
 </box>
 
-<box type="tip" header="#### Tips">
+<box type="tip" header="##### Tips">
 
-1. New clashing schedule will be informed so that you can modify using the [`edit` command](#editing-a-student--edit)
-2. <b>RATE</b> is the tuition fee per hour.
+* New clashing schedule will be informed so that you can modify using the [`edit` command](#editing-a-student--edit)
+* <b>RATE</b> is the tuition fee per hour.
     
 </box>
 
 ### Listing all students : `list`
 
-Shows a list of all students in the address book.
+Shows a list of all students in UG Teach.
 
 Format: `list`
 
 ### Editing a student : `edit`
 
-Edits an existing student in the address book.
+Edits an existing student in the UG Teach.
 
 **Format:** `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [r/RATE] [paid/PAID_AMOUNT] [owed/OWED_AMOUNT]`
 
 **Examples:**
 
-`edit 1 p/87438808 e/alexyeoh100@example.com` </br> Edits the phone number and email address of the 1st student to be `87438808` and `alexyeoh100@examnple.com` respectively.
+* `edit 1 p/87438808 e/alexyeoh100@example.com` edits the phone number and email address of the 1st student to be `87438808` and `alexyeoh100@examnple.com` respectively.
 
-`edit 2 paid/1200.00 owed/0` </br> Edits the paid amount of the 2nd student to be `$1200.00` and edits the owed amount to be `$0.00`.
+* `edit 2 paid/1200.00 owed/0` edits the paid amount of the 2nd student to be `$1200.00` and edits the owed amount to be `$0.00`.
 ![editResult.png](images/editResult.png)
```

Comment:
For this image, do you want to consider following the same as add command, like the Output:

Reply:
Thanks for the suggestion, I will change